xml/System.Web.Security/SqlMembershipProvider.xml

```diff
@@ -19,18 +19,20 @@
     <remarks>
       <format type="text/markdown"><![CDATA[  
   
-## Remarks  
- This class is used by the <xref:System.Web.Security.Membership> and <xref:System.Web.Security.MembershipUser> classes to provide membership services for an ASP.NET application using a SQL Server database. You cannot use a <xref:System.Web.Security.SqlMembershipProvider> without SQL Server. When your computer has SQL Server Express installed with the default instance name and user-instancing enabled, the <xref:System.Web.Security.SqlMembershipProvider> object will create a database called `aspnetdb` in the application's `App_Data` directory the first time the application is run.  
+## Remarks
+
+> [!NOTE]
+> In-box ASP.NET membership providers have been superseded by [ASP.NET Identity](/aspnet/identity/overview/getting-started/introduction-to-aspnet-identity). Please update applications to use the ASP.NET Identity platform instead of the in-box membership providers.
```

Comment:
```suggestion
> ASP.NET membership providers have been superseded by [ASP.NET Identity](/aspnet/identity/overview/getting-started/introduction-to-aspnet-identity). You should update your applications to use the ASP.NET Identity platform instead of the built-in membership providers.
```